store/tikv/region_request.go

```diff
@@ -101,6 +114,19 @@ func (ss *RegionBatchRequestSender) sendStreamReqToAddr(bo *Backoffer, ctxs []co
 	return
 }
 
+func recordRegionRequestRuntimeStats(stats map[tikvrpc.CmdType]*RegionRequestRuntimeStats, cmd tikvrpc.CmdType, d time.Duration) {
+	stat, ok := stats[cmd]
+	if !ok {
+		stats[cmd] = &RegionRequestRuntimeStats{
```

Comment:
Will this method be invoked parallelly in `BatchPointGet`?

Reply:
No, currently won't, `BatchPointGet `  will use `BatchGet` RPC, and each goroutine of `BatchGet` will has it own client, so this function won't be invoked parallelly.